tensorflow/core/kernels/mkl_quantize_op.cc

```diff
@@ -135,7 +401,29 @@ class MklQuantizeV2Op : public OpKernel {
         src_mkl_shape.IsMklTensor()
             ? src_mkl_shape.GetMklLayout()
             : memory::desc(src_dims, MklDnnType<float>(), dst_layout_type);
-    src.SetUsrMem(src_md, &src_tensor);
+
+    // If the mode is min_first, input data has to be subtracted from
+    // min_range, before being scaled
+    auto flat_input = input.flat<float>().data();
+    if (mode_ == QUANTIZE_MODE_MIN_FIRST) {
+      float* minfirst_input = GetMinfirstInputBuf(input.NumElements());
+      const Eigen::TensorOpCost cost(
+          sizeof(float), /*load bytes*/
+          sizeof(float), /*saved bytes*/
+          /*sub cost*/ Eigen::TensorOpCost::AddCost<float>());
```

Comment:
Nit: Can we make the comment placement consistent?
```suggestion
          Eigen::TensorOpCost::AddCost<float>() /*sub cost*/);
```

Reply:
Done.